include/RE/Misc/Calendar.h

```diff
@@ -73,6 +73,7 @@ namespace RE
 		std::string	  GetDayName() const;
 		std::uint32_t GetDayOfWeek() const;
 		float		  GetDaysPassed() const;
+		std::string   GetDisplayString(bool a_showYear);
```

Comment:
I wouldn't bother wrapping this, it's often desirable to have raw access to the actual function, especially if you're dealing with native types like `BSString`